osi3trace/osi_trace.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, path=None, type_name="SensorView", cache_messages=False):
         self.read_complete = False
         self.message_cache = {} if cache_messages else None
         self._header_length = 4
+        self.timestep_count = 0
```

Comment:
This is redundant to the length of message_offsets, and only valid once a file has been fully read in, so is not an apropriate API to the outside.